build.gradle.kts

```diff
@@ -149,4 +161,20 @@ if (signingPropsFile.exists()) {
             project.ext.set(key, value)
         }
     }
-}
\ No newline at end of file
+}
+
+android {
+    namespace = "ca.gosyer"
+
+    sourceSets["main"].manifest.srcFile("src/androidMain/AndroidManifest.xml")
+
+    defaultConfig {
+        compileSdk = 34
+        minSdk = 24
```

Comment:
What is the most viable minSdk possible? I am hoping for at least 21, but its fine if I can't go lower then 24.

Reply:
21 is feasible, setting 24 is because it was originally intended to use `Context#getDataDir()`, which requires a minimum of 24, but it is not used now.